.golangci.yml

```
@@ -211,7 +213,17 @@ issues:
   - linters:
     - revive
     - stylecheck
-    path: test/.*.go
+    path: _test\.go
+    text: should not use dot imports
+  - linters:
+    - revive
+    - stylecheck
+    path: (framework|e2e)/.*.go
```

```suggestion
    path: e2e/.*.go
```
there is no `framework` dir in CAPV, as compared to CAPI 
https://github.com/kubernetes-sigs/cluster-api/blob/main/.golangci.yml#L252